### ISSUE TITLE
monix-redis 0.6.0-RC-1 dependent on scalapb

### ISSUE THREAD

**Issue by VladKopanev** [monix/monix-connect#652]
We are trying to use `monix-redis` version 0.6.0-RC-1 in our project, but at the same time, our project is dependent on `scalapb-runtime`  version 0.9.4 which is binary incompatible with `scalapb-runtime` version 0.10.1 which is brought by `monix-redis` dependency, hence errors in runtime.
I think `monix-redis` should exclude any `scalapb` dependency because why have them?

**Comment by paualarco** [comment]
You are right, ScalaPB dependency should not be included in the `monix-redis` artifact. The dependency is just used in tests so we need to avoid that. 
For the time being, I think you could either force using `0.9.4` in your project or excluding the one that comes along with _monix_?

**Comment by VladKopanev** [comment]
Yeah, no problems, I just added `excludeAll(ExclusionRule(organization = "com.thesamet.scalapb")` to my `monix-redis` dependency and it works fine.

**Comment by paualarco** [comment]
Great!
